api/src/main/resources/workbench.yaml

```diff
@@ -3629,10 +3629,11 @@ definitions:
         type: string
       areaOfResearch:
         type: string
-      institutionalAffiliations:
+      deprecatedInstitutionalAffiliations:
```

Comment:
If we do a process like this, we shouldn't change wire format (model names are fine, but not property names like this)

Reply:
A couple of things can be marked deprecated on the swagger side as well https://swagger.io/docs/specification/describing-parameters/

I'm kinda thinking the path of least resistance is to come up with a different, shorter name for the new stuff,  and then just kill this one all at once. Maybe just Affiliation for the object, etc.